README.md

```diff
@@ -98,12 +98,30 @@ Just pick one of the open tickets. We will provide mentorship for all of them. :
 
 
 ## Developer guide
-
 This project uses [pipenv](https://docs.pipenv.org/) for managing the development environment. If you don't have it installed, run
 
 ```
 pip install pipenv
 ```
+After installing `pipenv`, enter the project directory and create virtual enviroment by running
+
+```
+pipenv shell
+```
```

Comment:
I would move the part until here to a separate "Troubleshooting" section:

> In case you want to test your changes within a virtual environment, run
> 
> ```
> pipenv shell
> ```

Reply:
I would also add a command for exiting virtualenv.

What is the preferred way: via `deactivate` or via `exit`?

Reply:
`exit`

Reply:
(deactivate was the virtualenv way)